File: codesignal/arcade/Intro/Csharp/allLongestStrings.cs
string[] allLongestStrings(string[] inputArray) {
    List<string> result = new List<string>();
    int maxLength = 0;
    for(int i = 0; i < inputArray.Length; i++){
        if(inputArray[i].Length > maxLength){
            result = new List<string>();
            maxLength = inputArray[i].Length;
            result.Add(inputArray[i]);
        }else if(inputArray[i].Length == maxLength){
            result.Add(inputArray[i]);
        }
    }
    
    return result.ToArray();
}

File: codesignal/arcade/Intro/JS/allLongestStrings.js
function allLongestStrings(inputArray) {
    var maxLength = 0
    for(let i = 0; i < inputArray.length; i++){
        if(maxLength < inputArray[i].length){
            maxLength = inputArray[i].length
        }
    }
    
    return inputArray.filter(function(value){
        return value.length == maxLength
    })
}
